.tools/runtime-upgrade/README.md

```diff
@@ -0,0 +1,29 @@
+# Runtime upgrades
+
+This tool is used by SRE for runtime upgrades. It provides two modes of runtime upgrading: `sudo` based and `democracy`-based.
+
+## Protocol
+
+The tool performs the following steps in sequence:
+
+1. Initialize a local chain using [`fork-of-substrate`](https://github.com/maxsam4/fork-off-substrate). TODO
+
+2. Performs a `sudo`-based runtime upgrade against the local chain to verify that the upgrade will succeed. TODO
+
+3. If 2 is successful, perform either a `sudo`-based upgrade or `democracy` proposal, depending on the provided flags. DONE
```

Comment:
@KaiserKarel Considering these should perform in sequence why are 1. and 2. marked TODO while 3. is marked DONE?
Is this intended or am I missing something?

Reply:
Yeah, got blocked on that; adding those features later as it is already usable

Reply:
work in progress by others. let them update.